pkg/sensors/accelerator/gpu/source/gpu_habana.go

```diff
@@ -0,0 +1,112 @@
+//go:build gpu && habana
```

Comment:
I think we can get away with just one of the tags rather than both enabled at the same time... ?

Reply:
esp when the `InitLib()` function for Habana is not doing anything